processor/src/memory/mod.rs

```diff
@@ -314,3 +330,56 @@ impl Default for Memory {
         Self::new()
     }
 }
+
+// MEMORY LOOKUPS
+// ================================================================================================
+
+/// Contains the data required to describe a memory read or write.
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+pub(super) struct MemoryLookup {
+    ctx: Felt,
+    addr: Felt,
+    clk: u64,
```

Comment:
Agreed - I actually started to use u32 throughout this PR but then switched because the AuxColumnBuilder is currently defined as usize. I think it makes sense to open an issue